packages/fluentui/accessibility/src/behaviors/Tree/treeTitleAsListItemTitleBehavior.ts

```diff
@@ -14,7 +17,8 @@ export const treeTitleAsListItemTitleBehavior: Accessibility<TreeTitleBehaviorPr
     attributes: {
       root: {
         ...(!props.hasSubtree && {
-          role: 'listitem',
+          role: props.selectable ? 'option' : 'listitem',
+          ...(props.selectable && { 'aria-selected': !!props.selected }),
```

Comment:
not sure what is going on, but in the final DOM there is aria-selected as well on the items which are not "selectable"
Please if you could try to fix it :)
![image](https://user-images.githubusercontent.com/22620150/93876371-0a591580-fcd7-11ea-8935-b3df0c3382f4.png)

Reply:
Just saw it also happens in the normal multi selectable tree

Reply:
Okay I will look into this